pkg/kubelet/rkt/rkt.go

```diff
@@ -937,15 +985,21 @@ func (r *Runtime) preparePod(pod *api.Pod, pullSecrets []api.Secret) (string, *k
 		newUnitOption("Service", "ExecStopPost", markPodFinished),
 		// This enables graceful stop.
 		newUnitOption("Service", "KillMode", "mixed"),
+		// Track pod info for garbage collection
+		newUnitOption(unitKubernetesSection, unitPodUID, string(pod.UID)),
```

Comment:
I chose to store it the systemd service file too since that's already found for Garbage Collect.  Also, the manifest file is removed from the tempdir after preparePod(), so it's not guaranteed to be around during GC, but the systemd unit will be.  Right?  Or is there another way to get at the pod manifest given the systemd unit file?